# Define msg for camera angle
int16 camera_angle